exporter/exporter.go

```diff
@@ -713,6 +713,10 @@ func aggregateMapValues(values []metricValue) []aggregatedMetricValue {
 	mapping := map[string]*aggregatedMetricValue{}
 
 	for _, value := range values {
+		// labels can be empty if they are filtered away by decoders
+		if len(value.labels) == 0 {
```

Comment:
This shouldn't happen at this point. I made https://github.com/cloudflare/ebpf_exporter/pull/479 to address this earlier in the pipeline.

Reply:
Thanks, LGTM